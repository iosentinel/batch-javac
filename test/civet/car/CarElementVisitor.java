package car;

import car.elements.Body;
import car.elements.Engine;
import car.elements.Wheel;

public interface CarElementVisitor {
  void visit(Wheel wheel);

  void visit(Engine engine);

  void visit(Body body);

  void visit(Car car);
}