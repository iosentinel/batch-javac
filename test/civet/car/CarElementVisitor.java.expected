package car;

import car.elements.Body;
import car.elements.Engine;
import car.elements.Wheel;

public interface CarElementVisitor {
    
    void visit(Wheel wheel);
    
    void visit(Engine engine);
    
    void visit(Body body);
    
    void visit(Car car);
    
    public void visit$1003();
    
    public void visit$1005();
    
    public void visit$1007();
    
    public void visit$1009();
    
    public void visit$1010();
}